README.md

```diff
@@ -4,12 +4,21 @@
 
 UI for managing recalls
 
-### Dependencies
+## Dependencies
 The app requires:
 * hmpps-auth - for authentication
 * redis - session store and token caching
 * manage-recalls-api - api service for managing the recall process
 
+### Node and NPM
+The required versions of Node.js and NPM are in package.json 'engines' field.
+Recommended that you use NVM to manage your Node versions. You can then upgrade to match the required versions with:
+```
+nvm install <version>
+nvm install-latest-npm
+```
+The Node version is also in .nvmrc so your shell can automatically switch to it when you change folders.
```

Comment:
What does this mean. Do we need to do anything in order to get our shell to switch automatically? (can/will)?

Reply:
Maybe I'll take this and the .nvmrc out. It's just a convenience, not required
Yes you'd need to add something to your shell config. I'll take this out until I can improve it.

Reply:
Removed it